arabic/learn/whattolearn/computer-science/computer-science.md

```diff
@@ -0,0 +1,4 @@
+لتعلم علوم الحاسب يمكنك اتباع المسار الاتي يوفر لك اهم الكورسات اللازمة دراستها
+ومصادر مجانية لدراسة تلك الكورسات
+
+- [مسار علوم الحاسب](https://github.com/open-source-society/computer-science#order-of-the-classes)
```

Comment:
This link seems to be an absolute link to GitHub rather than a relative link.  If you are linking to another page in the create-your-own-adventure repository, please use relative links to do so.  See the README of this repository for more information.